users/stdiscosrv.rst

```diff
@@ -68,6 +68,10 @@ list, e.g. ``https://disco.example.com:8443/``. Note that stdiscosrv uses port
 8443 by default. For stdiscosrv to be available over the internet with a dynamic
 IP address, you will need a dynamic DNS service.
 
+.. deprecated:: v0.14.44
+   Prior version need ``/v2/`` appended to the discovery
```

Comment:
I feel a bit silly about the amount of review going into such a little change (thanks already for your patience), but could do version->versions.

Reply:
It would have saved me 6 hours of time so it's relatively important I think, but it okay, I know that